Create a chapter on Book Dash (for the community handbook) (#1262)
alan-turing-institute/the-turing-way: book/website/community-handbook/bookdash/bookdash-application.md

(ch-bookdash-application)=
# Application and Review Process

The application, review, and selection process have been developed to provide a fair opportunity for all the interested members to show their interest in attending our book dash events.

In this subchapter, we have provided details on the book dash's application and reviewing process.

## Call for applications

The book dash event is organised for up to 15 participants.
The reason behind keeping it small is to allow the core team members to offer specific mentorship to the invited participants and make this an interactive, intimate, fun, and productive event.

To fairly make this selection, a call for applications is opened at least 3 months in advance so that interested members can share their skills, interest, and expected outcomes through a short application.
The applications are kept open for a minimum of 4 weeks and the timeline is communicated clearly through our newsletters, Twitter feed, and application form.

> These responses to the questions in the application form help us select a group of people who will be able to work effectively together and cover a broad set of possible contributions to the Turing Way. 

Author: ```suggestion
In 150 words each, applicants respond to the following questions:
- What could you contribute to the Turing Way?
- What would you gain from being part of the Turing Way book dash?

These responses to the questions in the application form help us select a group of people who will be able to work effectively together and cover a broad set of possible contributions to the Turing Way. 
```

We don't expect that the candidates might have previously contributed to the project or are familiar with our project infrastructure such as GitHub-based contribution. 
We look more for their ideas on what they think is important and helpful for others doing data science in the current scenario.

> In 150 words each, applicants respond to the following questions:
> - What could you contribute to the Turing Way?
> - What would you gain from being part of the Turing Way book dash?

Author: ```suggestion
```

(ch-bookdash-application)=
# Application and Review Process

> 

Reviewer: Given what is written below on rubrics for assessment, would it be worth including examples of the other questions applicants have to answer alongside accessibility and diversity info? If you would prefer not to disclose it that's fine too but might be worth adding a line for context.

Author: I think it is a good idea. We do have the application form public so this can be written transparently here.

Author: Added questions from the application: https://github.com/alan-turing-institute/the-turing-way/pull/1262/commits/9d1acb2a88132f353358b3878c6ed3881ce562bf

The application, review, and selection process have been developed to provide a fair opportunity for all the interested members to show their interest in attending our book dash events.

In this subchapter, we have provided details on the book dash's application and reviewing process.

> ## Call for application

Reviewer: Should be "Call for applications"?

> ## Call for application

Author: ```suggestion
## Call for applications
```

The book dash event is organised for up to 15 participants.
The reason behind keeping it small is to allow the core team members to offer specific mentorship to the invited participants and make this an interactive, intimate, fun, and productive event.

> To fairly make this selection, a call for application is opened at least 3 months in advance so that interested members can share their skills, interest, and expected outcomes through a short application.

Reviewer: Should be "call for applications"?

> To fairly make this selection, a call for application is opened at least 3 months in advance so that interested members can share their skills, interest, and expected outcomes through a short application.

Author: ```suggestion
To fairly make this selection, a call for applications is opened at least 3 months in advance so that interested members can share their skills, interest, and expected outcomes through a short application.
```

We also ask our applicants to indicate if they consider themselves to be a member of one or multiple historically underrepresented groups.
Here they can add any other information about themselves that they think the core team should know when reviewing their application (such as LGBTQ+, immigrant, disabled, parent, carer, POC members).

We seek to remove any barriers that might prevent our participants from attending the event.
Hence, we use this form to assess what support we can offer, including funding requirements, to our participants in ensuring that they can comfortably attend the event.
Applicants can mention their travel, accommodation, childcare, and accessibility requirements for their participation.

All these responses are securely handled under the [GDPR (General Data Protection Regulation) law](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation) (learn about creating [GDPR-proof surveys](https://www.surveylegend.com/gdpr/how-to-make-gdpr-proof-surveys-or-forms/)).

## Reviewing process

> 

Reviewer: Different numerical scoring systems referred to, (both 0,1,2 and 1,2,3) - best to pick one for consistency.

Applications for _The Turing Way_ book dashes are reviewed by at least 2 members of _The Turing Way_ core team.
A response form created by the organising members using the rubrics described below, the response from which are shared among all the reviewers during a designated meeting between the panel members.

> Scoring for each criterion is done between 0 to 2, where 2=criteria fully met, 1=criteria partially met, and 0=criteria not met.

Author: ```suggestion
Scoring for each criterion is done between 1 to 3, where 3=criteria fully met, 2=criteria partially met, and 1=criteria not met.
To avoid personal bias, each of these scores is clearly defined for every question in the review response form, which is explained below.
```

2. Applicants who score **mostly 2s** are enthusiastic if not wholly suitable for the project/event, for example they may come with some ideas for a contribution that is currently not in the scope of the project/event.
3. Applicants who score **mostly 3s** are clearly ready, goal-oriented, interested in contributing to the project, and excited to learn from others at the event.

Reviewers are also asked whether they would recommend an applicant. The available answers are "No", "Yes", "Unsure" or "Wild-Card/Red-Flag (request to discuss)".
Candidates who have ratings of "No" or "Unsure" across all reviewers will be rejected.
All other candidates will be discussed at the panel selection.

We ask all reviewers to finish the review with a few summary sentences to describe the application and the motivations for their scores to facilitate conversation during the selection panel discussion.
Finally, reviewers will also disclose if they have any conflict of interest that may influence their review leading to any bias in their decision.

> Here are the main areas that we will look into when reviewing an application:

Reviewer: Think these two sections (above and below this comment) duplicate so I would delete the one above and incorporate some of the contextual comments (i.e. people who are rejected by all reviewers are rejected outright) into the section below. The section below offers the most detail on the process through calling out the specific aspects reviewers are asked to rate people on.

Author: I need to think about this part. The part above this comment is in the introduction guide for the reviewers to explain the rationale of using this scoring system. The part below explains these scores for each question.

Maybe I need to explain that clearly to avoid the feeling of repeatition.

Author: Created a table to describe the criteria and avoid redundancy: https://github.com/alan-turing-institute/the-turing-way/pull/1262/commits/32c2637bfe375cde0ec34669d78df6794e17a9ee
